azure-pipelines.yaml

```diff
@@ -0,0 +1,43 @@
+name: Publish Docker Jenkins Images
+trigger:
+  branches:
```

Comment:
You can include -> `batch: true` after trigger like
```
trigger:
  batch: true
  branches:
    include:

```

Reply:
done

Reply:
done